server/app/models/volume.rb

```diff
@@ -39,4 +39,13 @@ def services
     self.grid.grid_services.where("service_volumes.volume_id" => self.id)
   end
 
+  def driver_for_node(host_node)
```

Comment:
The middleman is there since the same logic is needed in two places, `ServicePodSerializer` and in `VolumeSerializer`